src/components/swap/styled.tsx

```diff
@@ -22,25 +22,45 @@ export const PageWrapper = styled.div`
 `
 
 // Mostly copied from `AppBody` but it was getting too hard to maintain backwards compatibility.
-export const SwapWrapper = styled.main<{ isDark?: boolean }>`
+const SwapWrapperOuter = styled.main<{ isDark?: boolean }>`
   position: relative;
-  background: ${({ theme }) => theme.surface1};
-  border-radius: 24px;
-  border: 1px solid ${({ theme }) => theme.surface3};
-  padding: 8px;
-  padding-top: 12px;
-  box-shadow: ${({ isDark }) =>
-    `0px 0px 10px 0px rgba(252, 114, 255, ${isDark ? '0.08' : '0.04'}), 0px 40px 120px 0px rgba(252, 114, 255, ${
-      isDark ? '0.18' : '0.12'
-    })`};
   z-index: ${Z_INDEX.default};
+  border: 1px solid ${({ theme }) => theme.surface3};
   transition: transform 250ms ease;
+  border-radius: 24px;
+
+  &:before {
+    content: ' ';
+    display: flex;
+    position: absolute;
+    inset: 0;
+    transform: scale(1.1);
+    filter: blur(50px);
+    background-color: rgba(252, 114, 255, 0.075);
```

Comment:
this was with feedback from hayden/cal on getting the glow to be a bit more subtle, less bandy, and to fit the container a bit more naturally than a shadow so we moved to a scaled up blur